simulation_parameters/microbe_stage/biomes.json

```diff
@@ -3437,5 +3437,391 @@
         }
       }
     }
+  },
+  "patch_event_template_biome": {
```

Comment:
One tiny naming thing: I've been trying to move us towards consistent camelCase naming for internal names of stuff, so could you rename this and the meteor internal names to be like `patchEventTemplateBiome`?